stories/feedback/Banner.stories.js

```diff
@@ -16,6 +18,28 @@ export const DefaultBanner = withInfo()(() => (
   </Banner>
 ));
 
+export const EmptyBanner = withInfo()(() => (
+  <Banner status="muted" title="Sending Domains" onDismiss={action('Dismiss Clicked')}>
+    <p>
+      Sending domains are used to indicate who an email is from via the "From:" header. DNS records
+      can be configured for a sending domain, which allows recipient mail servers to authenticate
+      your messages.
+    </p>
+    <Banner.Action
+      variant="outline"
+      color="blue"
+      onClick={() => action('Sending Domain Documentation Clicked')}
+    >
+      Sending Domain Documentation
+    </Banner.Action>
+    <Banner.Media>
+      <Picture seeThrough>
+        <Picture.Image src={Image} />
+      </Picture>
+    </Banner.Media>
+  </Banner>
+));
+
```

Comment:
Can we add another story with a video, just to see if that all works as expected